### ISSUE TITLE
[Feature request] Avoid uploading the same image twice

### ISSUE THREAD

**Issue by Oremountainflorian** [humhub-contrib/gallery#54]
At the moment it's possible to upload the same image twice to a gallery. There should be a window asking "Do you want to overwrite the existing image?".

**Comment by MasterMindNET** [comment]
Good day, @Oremountainflorian!

How is it should be done?

Only by checking the image name and the image extenstion (format - like .png, .jpg, .jpeg) or only by image name?

If we need to find it out by some more specific logic, please let me know your thoughts!

Best regards,
Roman

**Comment by Oremountainflorian** [comment]
@MasterMindNET Yes  I guess checking image name and extension both is a good way (not case sensitive). I would be cool when a dialog box pops out in case the already existing image will be overwritten.
